### PR TITLE
add orbit method to read from list of paths

### PR DESCRIPTION
**Pull request by LiangJYu**
This PR adds a method to correlate an orbit file to SAFE zip from a list of paths (in addition to correlating within a directory containing orbit files).

### CODE REVIEW

**Comment by LiangJYu** [comment]
Only commits 6a90b7b and after are relevant to this PR. I don't know why the other commits are displayed. Apologies for any confusion.

**Comment by yunjunz** [comment]
> Only commits 6a90b7b and after are relevant to this PR. I don't know why the other commits are displayed. Apologies for any confusion.

Maybe you want to do a `git rebase` to get rid of those ?

File: src/sentinel1_reader/sentinel1_orbit_reader.py [hunk 1]
```diff
@@ -10,9 +50,9 @@ def get_swath_orbit_file(zip_path: str, orbit_dir: str):
         Path to Sentinel1 SAFE zip file. File names required to adhere to the
         format described here:
         https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions
-    orbit_dir : string
-        Path to directory containing orbit files. Orbit files required to adhere
-        to naming convention found here:
+    orbit_file_list: list[str]
+        List containing orbit path strings. Orbit files required to adhere to
```
Comment (vbrancat) [inline src/sentinel1_reader/sentinel1_orbit_reader.py:54]: path strings ---> file paths?

**Review by vbrancat — APPROVED** [review]
LGTM. Tested it and I did not see any issue with it.

**Review by yunjunz — APPROVED** [review]
I only have one minor suggestion: update the `sentinel1_orbit_reader.get_swath_orbit_file()` in the README usage to the new name.

The rest looks all good to me.

**Pull request merged**